plugins/node/opentelemetry-instrumentation-mongodb/README.md

```diff
@@ -55,6 +55,22 @@ Mongodb instrumentation has few options available to choose from. You can set th
 | `responseHook` | `MongoDBInstrumentationExecutionResponseHook` (function) | Function for adding custom attributes from db response |
 | `dbStatementSerializer` | `DbStatementSerializer` (function) | Custom serializer function for the db.statement tag |
 
+## Semantic Conventions
+
+This package uses `@opentelemetry/semantic-conventions` version `1.22+`, which implements Semantic Convention [Version 1.7.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.7.0/semantic_conventions/README.md)
+
+Attributes collected:
+
+| Attribute               | Short Description                                                              | Notes                                 |
+| ----------------------- | ------------------------------------------------------------------------------ | ------------------------------------- |
+| `db.system`             | An identifier for the database management system (DBMS) product being used.    | Key: `SEMATTRS_DB_SYSTEM`             |
```

Comment:
we will no longer add a column for notes and we will remove from the existing ones. So you can remove it from this PR to keep it consistent

Reply:
resolving as @david-luna addressed this in [5a71fb4](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2088/commits/5a71fb49ea4c19de218ee87fa9a473aa03b6ade6)